lib/msal-node/package.json

```diff
@@ -70,5 +70,8 @@
     "axios": "^0.21.1",
     "jsonwebtoken": "^8.5.1",
     "uuid": "^8.3.0"
+  },
+  "engines" : {
+    "node" : ">=14.0.0"
```

Comment:
I think it would be good to add to the readme detailing what are strategy is and which verisons are supported (i.e. a major version of MSAL node will support the active LTS releases and up at the time it was GA'd)

Reply:
One thing to consider: latest version of Azure Functions supports Node 10 and 12 https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions

Reply:
Given those are maintenance LTS, we may want to support those in v1.

Reply:
Sure. I was going with the active releases for `Node.js`. Given that there could be scripts dependent on these, we can start with `10.0.0` I suppose, since we already tested against `>=10.0.0`.

@tnorling  I no longer see version testing in `node` E2E tests. Did we revert the changes or am I checking the wrong script?

Reply:
Hasn't been merged yet. It's in Hector's PR #2684

Reply:
Updated the docs.